deploy/examples/create-external-cluster-resources.py

```diff
@@ -195,6 +195,8 @@ def gen_arg_parser(cls, args_to_parse=None):
                                   help="Ceph Manager prometheus exporter port")
         output_group.add_argument("--rbd-metadata-ec-pool-name", default="", required=False,
                                   help="Provides the name of erasure coded RBD metadata pool")
+        output_group.add_argument("--dry-run", default=False, action='store_true',
+                                  help="Dry run the python script")
```

Comment:
We need a better helper description. "Dry run prints the executed commands without running them"

Reply:
done